sphinx/builders/qthelp.py

```diff
@@ -300,9 +300,9 @@ def keyword_item(self, name, ref):
 
         if id:
             item = ' ' * 12 + '<keyword name="%s" id="%s" ref="%s"/>' % (
-                name, id, ref[1])
+                name, id, htmlescape(ref[1]))
```

Comment:
I noticed `name` is not escaped. double-quotes can break XML tag.
I withdraw this PR.